### PR TITLE
fix: vm-forward example

### LINKED ISSUE #59: vm_forward.nix example passes net option twice
Running `nixos-shell` on `examples/vm_forward.nix` results in the following error:

```
nixos-shell/examples: nixos-shell vm-forward.nix
trace: warning: system.stateVersion is not set, defaulting to 22.05. Read why this matters on https://nixos.org/manual/nixos/stable/options.html#opt-system.stateVersion.
/nix/store/ii93pa9vnfcf5j9niwglnf3m71yrm97j-nixos-vm
qemu-kvm: -netdev user,id=user.0,hostfwd=tcp::2222-:22: Duplicate ID 'user.0' for netdev
```

When looking at the corresponding `run-nixos-vm` script we see that qemu gets two `-net` and `-netdev` options, both with `id=user.0`.

```
-net nic,netdev=user.0,model=virtio -netdev user,id=user.0,"$QEMU_NET_OPTS" -net nic,netdev=user.0,model=virtio -netdev user,id=user.0,hostfwd=tcp::2222-:22
```

### PR DESCRIPTION
**Pull request by brianmcgee**
lib.mkForce is required

Closes #59

### CODE REVIEW

**Comment by Jaculabilis** [comment]
After getting this error and finding the issues here, I tried both `mkForce` and @fleaz's solution from #51 using `virtualisation.forwardPorts`. Both worked for what I tested, but I think the latter is a more user-friendly fix, since it uses existing NixOS options instead of the more heavy-handed `mkForce`.

File: examples/vm-forward.nix [hunk 1]
```diff
@@ -1,10 +1,9 @@
-{ pkgs, ... }: {
+{ pkgs, lib, ... }: {
```
Comment (Jaculabilis) [inline examples/vm-forward.nix:1]: Not needed now

**Comment by Mic92** [comment]
bors merge

**Comment by Mic92** [comment]
Thx!

**Pull request merged**